gym/core.py

```diff
@@ -385,7 +431,36 @@ def reward(self, reward):
 
 
 class ActionWrapper(Wrapper):
-    """A wrapper that can modify the action before :meth:`env.step`."""
+    """Superclass of wrappers that can modify the action before :meth:`env.step`.
+
+    If you would like to apply a function to the action before passing it to the base environment,
+    you can simply inherit from :class:`ActionWrapper` and overwrite the method :meth:`action` to implement
+    that transformation. The transformation defined in that method must take values in the base environment’s
+    action space. However, its domain might differ from the original action space.
+    In that case, you need to specify the new action space of the wrapper by setting :attr:`self._action_space` in
+    the :meth:`__init__` method of your wrapper.
+
+    Let’s say you have an environment with action space of type :class:`gym.spaces.Box`, but you would only like
+    to use a finite subset of actions. Then, you might want to implement the following wrapper::
+
+        class DiscreteActions(gym.ActionWrapper):
+            def __init__(self, env, disc_to_cont):
+                super().__init__(env)
+                self.disc_to_cont = disc_to_cont
+                self._action_space = Discrete(len(disc_to_cont))
```

Comment:
Should be `action_space` not `_action_space`

Reply:
`gym.Wrapper` actually has a property `def action_space` which returns `self._action_space`. I think this is the proper way to do it.